chunky/src/java/se/llbit/chunky/ui/render/tabs/TexturesTab.java

```diff
@@ -68,6 +68,7 @@ public void initialize(URL location, ResourceBundle resources) {
       editor.show();
     });
 
+    singleColorBtn.setTooltip(new Tooltip("Set block textures to a single color which is the average of all color values of its current texture. \nRequires restart for changes to take effect."));
```

Comment:
Does it _actually_ require a restart?

Reply:
It turns out that it does not require a restart. I'll remove the second line.